1.0-Upgrade.md

```diff
@@ -0,0 +1,106 @@
+# kubernetes-deploy 0.30.0 -> krane 1.0 migration guide
+
+**THIS MIGRATION GUIDE IS A WORK IN PROGRESS. THINGS CAN STILL CHANGE BEFORE 1.0**
+
+The `kubernetes-deploy` became `krane` in its 1.0 version, and besides the name change we also introduced new features and [breaking changes](CHANGELOG.md). This guide will help you transition to version 1.0 as smoothly as possible.
+
+**TL;DR**:
+* The command-line interface was redesigned; besides the name change, there are breaking changes in several flags.
+* The only breaking change in the public API is the renaming of the `KubernetesDeploy` namespace to `Krane`.
```

Comment:
This isn't true, the `template_dir`  is for sure gone, default values have changed too.